src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -78,6 +78,20 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
   </PropertyGroup>
 
+  <!-- Automatically reference NETStandard.Library or Microsoft.NETCore.App package if targeting the corresponding target framework.
+      We can refer here in the .props file to properties set in the .targets files because items and their conditions are
+      evaluated in the second pass of evaluation, after all properties have been evaluated. -->
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETStandard'">
+    <PackageReference Include="NETStandard.Library" Version="$(NetStandardDefaultPackageVersion)"/>
+  </ItemGroup>
+
+  <!-- The reference to the Microsoft.NETCore.App should generally not be surfaced to developers.  So mark it as
+       ReadOnly so it won't show updates to the package in the package manager UI, and don't allow disabling it
+       with the DisableImplicitFrameworkReferences property. -->
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
I don't like the inconsistency. Note that there is a real use case for not referencing this package at all: standalone app with smallest possible footprint. 

Also, it's worrying that the only clean way to consume a patch release would be to acquire a new SDK.